IDE/MonoDevelop/MonoDevelop.MonoGame/templates/MonoGameProject.xpt.xml

```diff
@@ -36,9 +36,23 @@
 				<Directory name="Content">
 					<File name="Content.mgcb" src="Common/Content.mgcb" BuildAction="MonoGameContentReference" />
 				</Directory>
+				<Directory name="x64">
+					<RawFile name="libopenal.so.1" src="libs/x64/libopenal.so.1" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
It did not enable `CopyToOutputDirectory` action, I'm guessing `RawFile` doesn't have it implement it or has it under different name? Anyway changing the tag to `File` should allow it to be copied out to output directory.